### ISSUE TITLE
Orange

### ISSUE THREAD

**Issue by clefebvre** [linuxmint/mint-y-colors#6]
**Old**

![image](https://user-images.githubusercontent.com/1138515/77822580-019ed280-70ec-11ea-85f1-780f721f08cb.png)

**New**

![image](https://user-images.githubusercontent.com/1138515/79365310-65c1f300-7f42-11ea-9187-4c169d024cf3.png)

**Comment by clefebvre** [comment]
<>

**Comment by kongreddgrass** [comment]
i like the new one, but we can do this more desaturated like #d6752f

**Comment by cheeesy** [comment]
I agree to @kongreddgrass, I think the old one is too desaturated and the new one is too saturated, I think #d6752f is a fair compromise.

**Comment by zboraon** [comment]
The new one is much better, the old one was too much pastel

**Comment by Nebelkind** [comment]
The new one can actually be called orange. Maybe a bit on the bright side, but definitely better than before.

**Comment by bluz71** [comment]
I don't like either.

However, I feel *new* is a better starting point, I would suggest backing off the brightness by at least 33%, maybe more.

**Comment by SebastJava** [comment]
![orange](https://user-images.githubusercontent.com/31940713/88553623-7975fe80-cff3-11ea-84cf-71632818ad8c.png)
That’s a though one. There is no way you could make a "calm" orange without making it brownish or beige. So i tried to make it just a bit "subdued" in an awkward manner: 90% saturation plus a bit of... lightening!? I just didn’t want to compromise on this one. :)
